handlers/eth1Token.go

```diff
@@ -118,7 +119,7 @@ func Eth1Token(w http.ResponseWriter, r *http.Request) {
 		Price:            template.HTML(fmt.Sprintf("<span>$%s</span><span>@ %.6f</span>", string(metadata.Price), ethExchangeRate)),
 	}
 
-	if handleTemplateError(w, r, eth1TokenTemplate.ExecuteTemplate(w, "layout", data)) != nil {
+	if handleTemplateError(w, r, "eth1Token.go / Eth1Token", eth1TokenTemplate.ExecuteTemplate(w, "layout", data)) != nil {
```

Comment:
This looks like a copy and paste error and should be
if handleTemplateError(w, r, "eth1Token.go / Eth1Token / Done", ...
to be consistent with your other changes.